shared/js/ui/templates/feedback-form.es6.js

```diff
@@ -16,7 +12,7 @@ module.exports = function () {
             <label class='frm__label'>Which website is broken?</label>
             <input class='js-feedback-url frm__input' type='text' placeholder='Copy and paste your URL' value='${this.model.url}'/>
             <label class='frm__label'>Describe the issue you encountered:</label>
-            <textarea class='frm__text js-feedback-message' placeholder='Which website content or functionality is broken? Please be as specific as possible.'></textarea>
+            <textarea class='frm__text js-feedback-message' required placeholder='Which website content or functionality is broken? Please be as specific as possible.'></textarea>
```

Comment:
Why this field is required and URL isn't?

Reply:
This form can also be used for standard feedback - without a broken URL.

Reply:
Makes sense 👍